drivers/net/mlx5/mlx5_rxtx.c

```diff
@@ -2242,6 +2243,51 @@ rxq_cq_to_ol_flags(struct rxq *rxq, volatile struct mlx5_cqe *cqe)
 	return ol_flags;
 }
 
+#define MLX5_VPMD_RXQ_RPLNSH_THRESH  (64U)
+
+/**
+ * Replenish buffers for RX in bulk.
+ * Backported from DPDK 17.11
+ *
+ * @param rxq
+ *   Pointer to RX queue structure.
+ * @param rel_data_offset
+ *   Should be 0 for ETH, GRH_HDR_LEN for IPoIB
+ */
+static inline void
+mlx5_rx_replenish_bulk_mbuf(struct rxq *rxq, signed int rel_data_offset) {
+	const uint16_t q_n = 1 << rxq->elts_n;
+	const uint16_t q_mask = q_n - 1;
+	uint16_t elts_idx = rxq->rq_ci & q_mask;
+	unsigned int i;
+
+	assert(rxq->refill_count >= MLX5_VPMD_RXQ_RPLNSH_THRESH);
+
```

Comment:
redundent line

Reply:
the assert? we took it from http://dpdk.org/browse/dpdk/tree/drivers/net/mlx5/mlx5_rxtx_vec.h?h=v17.11#n109